llvm/test/tools/sycl-post-link/virtual-functions/properties.ll

```diff
@@ -0,0 +1,42 @@
+; RUN: sycl-post-link -split=auto -properties -S < %s -o %t.table
+;
+; Device images with virtual functions in them should have
+; "virtual-functions-set" property under corresponding property set.
+; Device images that use virtual functions should have
```

Comment:
```suggestion
; Device images that use virtual functions should have the
```

Reply:
Applied in 9bef3a14ff2c8083378a19d7dd5a5c77d7ef1114